lib/streamlit/type_util.py

```diff
@@ -49,6 +49,9 @@
     from plotly.graph_objs import Figure
     from pydeck import Deck
 
+
+MAX_SF_DF_ROWS = 10000
```

Comment:
nit: I know I suggested this name, but I think we can go a bit more generic here so that we can reuse the same constant for other unevaluated / out-of-memory / out-of-core dataframe implementations. Just not sure about the name, maybe something like: `MAX_UNEVALUATED_DF_ROWS`. And maybe add a short comment saying something like: `Maximum number of rows to request from an unevaluated (out-of-core) dataframe`

Reply:
It's updated 👍